### PR TITLE
feat: Toggle drop sync server setting

### PR DESCRIPTION
**Pull request by Merlin6019**

### CODE REVIEW

**Comment by croscas** [comment]
Can you update the description of the PR with meaningful information? For example what problem is it solving, the changes required in the codebase and how have you tested that this change works as expected.

**Comment by RobbeBryssinck** [comment]
The idea is good, but there are some implementation problems. This is rather overkill. We're sending the option to the client here, but that seems unnecessary. The client doesn't really have to do anything for this behavior to function. Furthermore, we don't really need to store it in ServerSettings in this case. It can just stand on its own. Look at fGoldLossFactor for a better example.

File: Code/server/Services/InventoryService.cpp [hunk 1]
```diff
@@ -10,6 +10,13 @@
 #include <Messages/RequestEquipmentChanges.h>
 #include <Messages/NotifyEquipmentChanges.h>
 #include <Messages/DrawWeaponRequest.h>
+#include <client/World.cpp>
```
Comment (RobbeBryssinck) [inline Code/server/Services/InventoryService.cpp:13]: Same thing, don't include cpp.

File: Code/server/Services/InventoryService.cpp [hunk 2]
```diff
@@ -36,7 +41,11 @@ void InventoryService::OnInventoryChanges(const PacketEvent<RequestInventoryChan
     NotifyInventoryChanges notify;
     notify.ServerId = message.ServerId;
     notify.Item = message.Item;
-    notify.Drop = message.Drop;
+
+    if (bEnableItemDrops)
```
Comment (yamashi) [inline Code/server/Services/InventoryService.cpp:45]: These 4 lines could just be

notify.Drop = bEnableItemDrops ? message.Drop : false;

**Review by yamashi — CHANGES_REQUESTED** [review]

**Pull request merged**